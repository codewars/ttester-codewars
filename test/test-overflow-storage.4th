\ Copyright 2023 nomennescio

\ pass
<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e
->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e
}>

\ fail
<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
-> }>

<{
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
-> }>

<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
-> }>

<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
}>

<{
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
->
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
}>

<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
}>

<{
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
}>

<{
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
->
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
}>

<{ ->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
}>

<{ ->
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
}>

<{ ->
0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2  3  4  5  6  7  8  9  0  1  2
0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e 3e 4e 5e 6e 7e 8e 9e 0e 1e 2e
}>
