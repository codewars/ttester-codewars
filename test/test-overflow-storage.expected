
<PASSED::>Test Passed

<FAILED::>Too many cell results to test

<FAILED::>Too many float results to test

<FAILED::>Too many cell results to test
<:LF:>Too many float results to test

<FAILED::>Too many cell results to test

<FAILED::>Too many float results to test

<FAILED::>Too many cell results to test
<:LF:>Too many float results to test

<FAILED::>Too many cell results to test

<FAILED::>Too many float results to test

<FAILED::>Too many expected cell results to test

<FAILED::>Too many expected float results to test

<FAILED::>Too many expected cell results to test
<:LF:>Too many expected float results to test
